packages/yew/src/html/component/lifecycle.rs

```diff
@@ -105,44 +179,33 @@ impl<COMP: BaseComponent> Runnable for CreateRunner<COMP> {
     }
 }
 
-pub(crate) enum UpdateEvent<COMP: BaseComponent> {
-    /// Wraps messages for a component.
-    Message(COMP::Message),
-    /// Wraps batch of messages for a component.
-    MessageBatch(Vec<COMP::Message>),
+pub(crate) enum UpdateEvent {
+    /// Drain messages for a component.
+    Message,
     /// Wraps properties, node ref, and next sibling for a component.
-    Properties(Rc<COMP::Properties>, NodeRef, NodeRef),
+    Properties(Rc<dyn Any>, NodeRef, NodeRef),
```

Comment:
If the messages don't have to show up in the type anymore and are stored elsewhere, why not give the `Properties` a similar treatment instead of casting through `dyn Any`?

Reply:
It's not worth it unless down casting is significantly slower than mutably borrowing.